website/app/routes/show.js

```diff
@@ -141,6 +148,11 @@ export default class ShowRoute extends Route {
           hasSidecar,
           relatedComponents,
         };
+
+        // Cache the model
+        this.modelCache[path] = model;
+
+        return model;
```

Comment:
see suggestion above
```suggestion
        return this.modelCache[path];
```